docs/dxp/7.x/en/site-building/05-personalizing-site-experience/introduction-to-personalizing-site-experience.md

```diff
@@ -17,11 +17,11 @@ See [Creating and Managing a Segment] to get started.
 
 ### Advanced and Segmentation with Liferay Analytics Cloud
 
-Supercharge your Segments by integrating your Liferay DXP instance with Anaytics Cloud. Analytics Cloud enables you to integrate DXP with Salesforce(tm) and other data sources to provide even more ways to understand and segment your user base.
+Supercharge your Segments by integrating your Liferay DXP instance with Liferay Analytics Cloud. Analytics Cloud enables you to integrate DXP with Salesforce(tm) and other data sources to provide even more ways to understand and segment your user base.
 
 <!-- screenshot of AC view -->
 
-Integration with AC also provides in-depth (ML powered?) data analytics capabilities to help you understand user behavior and session data for every page of your DXP site.
+Integration with Analytics Cloud also provides in-depth (ML powered?) data analytics capabilities to help you understand user behavior and session data for every page of your DXP site.
```

Comment:
@jrhoun what is ML powered?

Reply:
Me wondering if there's any machine learning elements on the Analytics Cloud side that are in use to help folks analyze / understand the data they get from integration. I don't know if that's the case, but thought it'd be a relevant place to mention that IF there is.

Reply:
Ah, ok. I'll try to get some more information on that.

Reply:
I found some webinars on the subject, so it looks like you can use Machine Learning with analytics and personalization. I'm not sure how fully integrated that is though.